src/ssl.c

```diff
@@ -45596,7 +45596,10 @@ int wolfSSL_RSA_private_decrypt(int len, const unsigned char* fr,
     return ret;
 }
 
-#if !defined(_WIN32) && !defined(HAVE_FIPS) && !defined(HAVE_SELFTEST)
+#if defined(OPENSSL_EXTRA) && !defined(NO_RSA) && defined(WOLFSSL_KEY_GEN) &&\
```

Comment:
I don't think we need WOLFSSL_KEY_GEN protection here.  api.c includes that, since it does RSA key generation in the test function, but wolfSSL_RSA_public_decrypt() doesn't do key generation internally.